src/DIRAC/WorkloadManagementSystem/Service/JobManagerHandler.py

```diff
@@ -86,6 +86,9 @@ def initializeHandler(cls, serviceInfoDict):
     return S_OK()
 
   def initialize(self):
+    return self.initializeRequest()
```

Comment:
I think this needs to be moved from `JobManagerHandlerMixin` to `JobManagerHandler` as the docs state:

> You **should not** write a method called `initialize` because Tornado already use that name, so the initialize from DISET handlers became `initializeRequest`